### PR TITLE
Update the performance-timeline IDL file + test

### PR DESCRIPTION
**Pull request by lukebjerring**
Hello reviewer(s),

This PR is intended to consolidate the spec’s IDL definition with the WPT test suite’s copy, and any idlharness tests.

The up-to-date copy of the IDL file was automatically extracted from the reffy-reports repo (https://github.com/tidoust/reffy-reports/tree/master/whatwg/idl) which scrapes known specs automatically + regulary.

This PR is part of a migration project which will eventually be automatically updating and creating PRs for changes in spec IDL.

Please check that:
The spec (and its source) is correct and up-to-date
All tests which cover the IDL in the spec have been migrated to fetch + use the idl in the `interfaces/` directory (instead of inline copies in the test files).

### CODE REVIEW

File: performance-timeline/idlharness.any.js [hunk 1]
```diff
@@ -6,21 +6,22 @@
...
-  idl_array.add_objects({
-    Performance: ["performance"],
-    PerformanceMark: [self.performance.getEntriesByName("mark")[0]],
-  });
-  idl_array.test();
-}, "Test IDL implementation of performance-timeline API");
+    idl_array.add_objects({
+      Performance: ['performance'],
+      // PerformanceEntry implicitly tested in user-timing.
+      ResizeObserverEntry: ['observerEntry'],
```
Comment (lukebjerring) [inline performance-timeline/idlharness.any.js]: @npm1 - I'd like to cover `PerformanceObserverEntryList` but I'm not sure where to get an instance.
Am I missing something, or is it the same method set as `PerformanceEntryList`, so probably implemented by `Performance`, but not stated as such in the IDL?

Comment (npm1) [reply performance-timeline/idlharness.any.js]: PerformanceObserverEntryList is the first parameter that a callback to the PerformanceObserver receives. So you can get such an object by creating a PerformanceObserver and calling observe().

It is not quite the same as PerformanceEntryList. PerformanceObserverEntryList is a wrapper interface that allows calling getEntries() and a couple other methods, whose return values are a sequence of entries, i.e. a PerformanceEntryList.

Comment (igrigorik) [reply performance-timeline/idlharness.any.js]: @lukebjerring is this PR blocked on the above, or?

**Review by igrigorik — APPROVED** [review]
👍

**Pull request merged**